### DISCUSSION TITLE
When I autodownload the "latest" mongo with mongo memory server, I get v6.1.0-alpha-650-gc088640

### DISCUSSION THREAD

**Discussion by lagerspetz-starship** [typegoose/mongodb-memory-server#644] [Q&A] [answered]
When I autodownload the "latest" mongo with mongo memory server, it uses this url: https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1804-latest.tgz

That downloads v6.1.0-alpha-650-gc088640, which is not latest and not a stable version. Manually downloading the current latest stable version gives me 5.0.8. Could you fix this?

**Comment by hasezoey** [comment, marked as answer]
> Could you fix this?

it is not really a problem on the side of MMS, MMS just combines the version provided by the user (in your case `latest`) and puts it in the correct spot, if you want a stable version use `v5.0-latest`, see [this page](https://dl.mongodb.org/dl/linux) for more information on what is available (not that page can take a while to load)

**Reply by lagerspetz-starship** [reply]
Ok, thanks. In the end, I fixed the version via MONGOMS_VERSION=<version_number_here> as an env var.
